tags/1.2/_index.md

```diff
@@ -0,0 +1,40 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta Standard Tag Library Spec 1.2"
+date: 2019-07-18
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Standard Tag Library encapsulates as simple tags the core functionality common to many Web applications.
+Jakarta Standard Tag Library has support for common, structural tasks such as iteration and conditionals, tags
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./tags-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./tags-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
+* [Jakarta Standard Tag Library 1.2 Tagdoc](./tagdocs)
+* [Jakarta Standard Tag Library TCK](https://download.eclipse.org/jakartaee/tags/1.2/eclipse-tags-tck-1.2.0.zip)
+* Maven coordinates
+  * [jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:jar:1.2](https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/1.2/jar)
```

Comment:
Shouldn't this be version 1.2.5 or 1.2.7 (not sure what happened to 1.2.6):  https://oss.sonatype.org/content/repositories/staging/jakarta/servlet/jsp/jstl/jakarta.servlet.jsp.jstl-api/

Reply:
It should be 1.2.5 as the Jakarta EE 8 release with jakarta coords was the 1.2.4 release.

Reply:
We decided on the Spec Committee call that we will accept 1.2.7 version.  Please update this API coordinate to point at the 1.2.7 API jar file.

Reply:
Yes, @ren-zhijun-oracle in two places on the line above, the "1.2" should be "1.2.7".